src/App/Controller/SecurityController.php

```diff
@@ -4,28 +4,84 @@
 
 namespace App\Controller;
 
+use App\Security\User;
+use L3l0Labs\SystemAccess\Email;
+use L3l0Labs\SystemAccess\UseCase\RegisterUser\Command as RegisterUserCommand;
+use App\Form\RegisterUserType;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
 use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
 
 class SecurityController extends Controller
 {
     public function login(Request $request, AuthenticationUtils $authUtils): Response
     {
-        // get the login error if there is one
         $error = $authUtils->getLastAuthenticationError();
 
-        // last username entered by the user
         $lastUsername = $authUtils->getLastUsername();
 
-        return $this->render('security/login.html.twig', array(
+        return $this->render('security/login.html.twig', [
             'last_username' => $lastUsername,
-            'error'         => $error,
-        ));
+            'error'         => $error
+        ]);
     }
 
     public function logout()
     {
     }
+
+    public function register(Request $request)
+    {
+        $form = $this->createForm(RegisterUserType::class);
+        $form->handleRequest($request);
+
+
+
+        if($form->isValid()) {
```

Comment:
Form


Calling isValid() on a Form instance before submitting it
is deprecated and will throw an exception in Symfony 4.0.


Before:

   if ($form->isValid()) {
       // ...
   }
After:

   if ($form->isSubmitted() && $form->isValid()) {
       // ...
   }